### ISSUE TITLE
This is not full version of FoundationPress, when I installed it then I don't see any style from foundation.css. Where is css and scss folder and other...

### ISSUE THREAD

**Issue by vercic** [olefredrik/FoundationPress#711]
This is not full version of FoundationPress, when I installed it then I don't see any style from foundation.css. Where is css and scss folder and other...????

**Comment by vercic** [comment]
Can you put please the full version of FoundationPress will all last updates? Thank you.

**Comment by annedorko** [comment]
What steps did you follow to install FoundationPress? Have you run: `npm run watch` or `npm run build`?

**Comment by vercic** [comment]
I didn't use any commands. For me it's more easy to copy files of foundation press into my new folder on localhost and lunch the theme.. How is possible to have all files?

**Comment by annedorko** [comment]
FoundationPress isn't just a plain theme for copy/pasting, it's built with tools that speed up the development process from the cli, using the `npm run` commands I mentioned earlier while you develop.

When your theme is ready for production, run `npm run package` and use that .ZIP folder to grab the plain version of the theme without all the rest of the tools. It compiles everything for you into a packaged and upload-able theme like you're asking for.

TL;DR: To get FoundationPress to work you will need to follow the [installation guide](https://github.com/olefredrik/FoundationPress#quickstart).

**Comment by vercic** [comment]
Thank you. How to copy the site after to the server to put it online?

**Comment by annedorko** [comment]
Once you have run `npm run package`, go into the /packaged/ folder in the project, and you'll find the zipped folder for uploading to your WordPress installation.

**Comment by vercic** [comment]
Ok, will do it. Thank you very much for answer.
